jest.test.config.js

```diff
@@ -1,3 +1,29 @@
+// modules to be transpiled by Jest (using babel) as they come as ESM
+const esmModules = [
+  'comma-separated-tokens',
+  'decode-named-character-reference',
+  'fault',
+  'hast-util-whitespace',
+  'mdast-util-definitions',
+  'mdast-util-from-markdown',
+  'mdast-util-frontmatter',
+  'micromark',
+  'rehype-react',
+  'remark-frontmatter',
+  'remark-parse',
+  'remark-rehype',
+  'space-separated-tokens',
+  'trim-lines',
+  'unified',
+  'unist-util-filter',
+  'unist-util-generated',
+  'unist-util-is',
+  'unist-util-position',
+  'unist-util-stringify-position',
+  'unist-util-visit',
+  'vfile-message',
```

Comment:
It's unfortunate that we have to maintain the list of ESM only modules in a different place because of jest but it's still better than to not be able to upgrade them at all.

Reply:
There might be a better and more elegant solution, i didn't dig too much into jest. This solution seems to be working for now.